src/backend/distributed/commands/truncate.c

```diff
@@ -228,7 +228,7 @@ static void
 AcquireDistributedLockOnRelations(List *relationIdList, LOCKMODE lockMode)
 {
 	Oid relationId = InvalidOid;
-	List *workerNodeList = ActivePrimaryNodeList(NoLock);
+	List *workerNodeList = ActivePrimaryWorkerNodeList(NoLock);
```

Comment:
should this include the coordinator? I assume the truncate also happens there?

Reply:
Oh, yes, it is true.